trin-core/src/portalnet/metrics.rs

```diff
@@ -54,4 +78,24 @@ impl OverlayMetrics {
     pub fn report_inbound_offer(&self) {
         self.inbound_offer.inc();
     }
+
+    pub fn report_outbound_ping(&self) {
+        self.outbound_ping.inc();
+    }
+
+    pub fn report_outbound_find_nodes(&self) {
+        self.outbound_find_nodes.inc();
+    }
+
+    pub fn report_outbound_find_content(&self) {
+        self.outbound_find_content.inc();
+    }
+
+    pub fn report_outbound_offer(&self) {
+        self.outbound_offer.inc();
+    }
+
+    pub fn report_new_node_in_routing_table(&self) {
```

Comment:
Idk if this is the best approach? I think I'd rather see an accurate count of how many nodes are currently in my routing table. So, we should also decrement the count whenever a node is evicted. Otherwise this number will just continue to rise on-and-on every time a node is added

Reply:
@njgheorghita Good point, that should have been labelled something like "nodes seen". But "nodes connected" is much more useful, so I'm adding that as a Prometheus Gauge type instead of a counter. Currently debugging why it's telling me I have -2 connected nodes in the routing table.